docs/howto/manage-controllers.md

```diff
@@ -508,13 +530,11 @@ scrape_configs:
 (back-up-a-controller)=
 ## Back up a controller
 
-
-```{caution}
-The procedure documented below is currently supported only for machine (non-Kubernetes) controllers.
-```
 (create-a-controller-backup)=
 ### Create a controller backup
 
+> Only supported machine (non-Kubernetes) controllers.
```

Comment:
Is a blockquote best here?

Reply:
I don't have a strategy for how to supersede the blockquote for non-See content. I'll tackle these other cases in a separate PR.